### ISSUE TITLE
[BUG] Can't download support bundle on app mode

### ISSUE THREAD

**Issue by futuretea** [harvester/harvester#836]
**Describe the bug**

![image](https://user-images.githubusercontent.com/15064560/117428494-b7c42500-af58-11eb-9c6d-6f851510ed8d.png)

**To Reproduce**
1. Create a cluster by kubeadm
2. Apply harvester helm chart
3. Genereate support bundle

**Expected behavior**

Can genereate support bundle and download

**Support bundle**

**Environment:**
 - Harvester ISO version:  No ISO, app mode
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630):

**Additional context**
```bash
 # kubectl logs -f supportbundle-manager-bundle-pjvcf-7fd75b8f8-dftwc
+ exec tini -- /usr/bin/support-bundle-utils manager
time="2021-05-07T09:13:09Z" level=debug msg="generating cluster bundle..."
W0507 09:13:09.590675       8 warnings.go:70] batch/v1beta1 CronJob is deprecated in v1.21+, unavailable in v1.25+; use batch/v1 CronJob
time="2021-05-07T09:13:18Z" level=debug msg="file longhorn-support-bundle_becae30c-b225-46e4-b64e-5c44b7c48a64_2021-05-07T09-13-13Z.zip is downloaded to /tmp/harvester-support-bundle/bundle/external"
time="2021-05-07T09:13:18Z" level=error msg="no Harvester nodes are found"
time="2021-05-07T09:13:18Z" level=info msg="support bundle /tmp/harvester-support-bundle/harvester-supportbundle_650204af-e98d-4af4-a111-f01ab84c23ce_2021-05-07T09-13-09Z.zip ready for downloading"
```

```bash
❯ kubectl get no master -o yaml | kubectl neat
apiVersion: v1
kind: Node
metadata:
  annotations:
    csi.volume.kubernetes.io/nodeid: '{"driver.longhorn.io":"master"}'
    flannel.alpha.coreos.com/backend-data: '{"VNI":1,"VtepMAC":"3e:1e:7f:44:eb:06"}'
    flannel.alpha.coreos.com/backend-type: vxlan
    flannel.alpha.coreos.com/kube-subnet-manager: "true"
    flannel.alpha.coreos.com/public-ip: 192.168.1.213
    kubeadm.alpha.kubernetes.io/cri-socket: /var/run/dockershim.sock
    kubevirt.io/heartbeat: "2021-05-07T09:28:00Z"
    node.alpha.kubernetes.io/ttl: "0"
    volumes.kubernetes.io/controller-managed-attach-detach: "true"
  labels:
    beta.kubernetes.io/arch: amd64
    beta.kubernetes.io/os: linux
    cpu-feature.node.kubevirt.io/aes: "true"
    cpu-feature.node.kubevirt.io/arat: "true"
    cpu-feature.node.kubevirt.io/arch-capabilities: "true"
    cpu-feature.node.kubevirt.io/avx: "true"
    cpu-feature.node.kubevirt.io/f16c: "true"
    cpu-feature.node.kubevirt.io/fsgsbase: "true"
    cpu-feature.node.kubevirt.io/hypervisor: "true"
    cpu-feature.node.kubevirt.io/ibpb: "true"
    cpu-feature.node.kubevirt.io/invtsc: "true"
    cpu-feature.node.kubevirt.io/pcid: "true"
    cpu-feature.node.kubevirt.io/pclmuldq: "true"
    cpu-feature.node.kubevirt.io/popcnt: "true"
    cpu-feature.node.kubevirt.io/rdrand: "true"
    cpu-feature.node.kubevirt.io/rdtscp: "true"
    cpu-feature.node.kubevirt.io/rsba: "true"
    cpu-feature.node.kubevirt.io/skip-l1dfl-vmentry: "true"
    cpu-feature.node.kubevirt.io/smep: "true"
    cpu-feature.node.kubevirt.io/spec-ctrl: "true"
    cpu-feature.node.kubevirt.io/ss: "true"
    cpu-feature.node.kubevirt.io/sse4.2: "true"
    cpu-feature.node.kubevirt.io/stibp: "true"
    cpu-feature.node.kubevirt.io/tsc-deadline: "true"
    cpu-feature.node.kubevirt.io/tsc_adjust: "true"
    cpu-feature.node.kubevirt.io/vme: "true"
    cpu-feature.node.kubevirt.io/x2apic: "true"
    cpu-feature.node.kubevirt.io/xsave: "true"
    cpu-model.node.kubevirt.io/Nehalem: "true"
    cpu-model.node.kubevirt.io/Nehalem-IBRS: "true"
    cpu-model.node.kubevirt.io/Opteron_G1: "true"
    cpu-model.node.kubevirt.io/Opteron_G2: "true"
    cpu-model.node.kubevirt.io/Penryn: "true"
    cpu-model.node.kubevirt.io/Westmere: "true"
    cpu-model.node.kubevirt.io/Westmere-IBRS: "true"
    cpumanager: "false"
    hyperv.node.kubevirt.io/base: "true"
    hyperv.node.kubevirt.io/frequencies: "true"
    hyperv.node.kubevirt.io/reset: "true"
    hyperv.node.kubevirt.io/runtime: "true"
    hyperv.node.kubevirt.io/synic: "true"
    hyperv.node.kubevirt.io/synic2: "true"
    hyperv.node.kubevirt.io/synictimer: "true"
    hyperv.node.kubevirt.io/time: "true"
    hyperv.node.kubevirt.io/vpindex: "true"
    kubernetes.io/arch: amd64
    kubernetes.io/hostname: master
    kubernetes.io/os: linux
    kubevirt.io/schedulable: "true"
    node-role.kubernetes.io/control-plane: ""
    node-role.kubernetes.io/master: ""
    node.kubernetes.io/exclude-from-external-load-balancers: ""
  name: master
spec:
  podCIDR: 10.244.0.0/24
  podCIDRs:
  - 10.244.0.0/24
```
```bash
# kubectl get no
NAME     STATUS   ROLES                  AGE     VERSION
master   Ready    control-plane,master   6h31m   v1.21.0
```

**Comment by guangbochen** [comment]
@bk201 I think we should still be able to generate and download the support bundle file even if the node logs can't be collected due to different environment setups, thanks.

**Comment by bk201** [comment]
This is also addressed in https://github.com/harvester/support-bundle-utils/pull/6
The web server is not tied to the node bundle collecting step, it's up until the application is terminated.

**Comment by juadk** [comment]
Tested successfully in build `master-8cfc827f-head `​with dev-mod.
The progress field reported the progress as expected.

![support_bundle_dev_mode](https://user-images.githubusercontent.com/6025636/120981382-e9a6f200-c777-11eb-8fbb-b6d126ce0d4d.png)

I downloaded the file `harvester-supportbundle_59aa587d-9348-4fab-b996-43b75de772c6_2021-06-07T07-53-52Z.zip` and checked the content conformity. All looked good.
[harvester-supportbundle_59aa587d-9348-4fab-b996-43b75de772c6_2021-06-07T07-53-52Z.zip](https://github.com/harvester/harvester/files/6607098/harvester-supportbundle_59aa587d-9348-4fab-b996-43b75de772c6_2021-06-07T07-53-52Z.zip)
